hydrolib/core/dflowfm/xyz/parser.py

```diff
@@ -22,13 +22,20 @@ def parse(filepath: Path) -> Dict:
         data: Dict = dict(points=[])
 
         with filepath.open() as f:
-            for line in f.readlines():
+            for linenr, line in enumerate(f.readlines()):
 
-                if len(line) < 5:  # 3 values, two whitespaces
+                line = line.strip()
+                if line.startswith("*") or len(line) == 0:
                     continue
 
-                x, y, z, c = re.split(xyzpattern, line, maxsplit=3)
+                try:
+                    x, y, z, *c = re.split(xyzpattern, line, maxsplit=3)
+                except ValueError:
```

Comment:
Can you add documentation to this method to indicate that this method may throw a `ValueError`?

Reply:
Done in new commit.